contracts/eternal-storage/OwnedEternalStorage.sol

```diff
@@ -0,0 +1,65 @@
+pragma solidity 0.5.2;
+
+import "./EternalStorage.sol";
+
+
+/// @title Owned Contracts
+/// @dev This contract provides ownership and access control functionality.
+/// Using the `onlyOwner` modifier, a function can be restricted to being
+/// called by the owner of the contract.  The owner of a contract can
+/// irrevocably transfer ownership using the `transferOwnership` function.
+contract Owned {
+    // Address of the owner of the contract
+    address private _owner;
+
+    /**
+     * @dev Access check: revert unless `msg.sender` is the owner of the contract.
+     */
+    modifier onlyOwner() {
+        require(msg.sender == _owner);
+        _;
+    }
+
+    /**
+     * @param owner The owner of the contract.  If set to `address(0)`, then
+     * `msg.sender` will be used instead.
+     */
+    constructor(address owner) public {
+        if (owner != address(0)) {
+            _owner = owner;
+        } else {
+            _owner = msg.sender;
+        }
+    }
+
+    /**
+     * @dev Allows the current owner to irrevocably transfer control of the
+     * contract to a `_newOwner`.
+     * @param _newOwner The address to transfer ownership to.
+     */
+    function transferOwnership(address _newOwner) external onlyOwner {
+        require(_newOwner != address(0));
+        emit OwnershipTransferred(_owner, _newOwner);
+        _owner = _newOwner;
+    }
+
+    /**
+     * @dev Returns the owner of the contract.
+     */
+    function getOwner() public view returns(address) {
+        return _owner;
+    }
+
+    /**
+     * @dev This event will be emitted every time the ownership of this
+     * contract changes.
+     * @param previousOwner representing the previous owner of the contract
+     * @param newOwner representing the new owner of the contract
+     */
+    event OwnershipTransferred(address indexed previousOwner, address indexed newOwner);
+}
+
+
+contract OwnedEternalStorage is Owned, EternalStorage {
+    constructor() public Owned(address(0)) {}
+}
```

Comment:
It's not important but the ending newline is missing here.

Reply:
Fixed.